### PR TITLE
Quality: PowerShell Summit GH-3375

### LINKED ISSUE #3375: Quality: Install-AdcsNetworkDeviceEnrollmentService
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples, Formatting command syntax

### Article List

- https://github.com/MicrosoftDocs/windows-powershell-docs/blob/main/docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md

### PR DESCRIPTION
**Pull request by Snozzberries**
# PR Summary

 - Resolves #3375

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md [hunk 1]
```diff
@@ -98,9 +135,10 @@ Accept wildcard characters: False
 ```
 
 ### -CAConfig
-Specifies remote certification authority (CA) that the Network Device Enrollment Service uses.
-This parameter is mandatory when used within the *ApplicationPoolIdentity* parameter.
-Do not use this parameter when a local CA is installed.
+
+Specifies remote certification authority (CA) that the Network Device Enrollment Service uses. This
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md:139]:
```suggestion
Specifies the remote certification authority (CA) that the Network Device Enrollment Service uses. This
```

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md [hunk 2]
```diff
@@ -130,12 +169,13 @@ Accept wildcard characters: False
...
 
 ### -Credential
-Specifies a **PSCredential** object that this cmdlet use to connect to the NDES role service.
-To obtain a credential object, use the **Get-Credential** cmdlet.
-For more information, type `Get-Help Get-Credential`.
-The NDES must be installed on a server that is a member of an Active Directory Domain Services (AD DS) domain.
-If NDES is configured to use a Standalone CA, then an account that is a member of the local Administrators on the CA is required.
-If NDES is installed to use an Enterprise CA, then using an account that is a member of Domain Admins group is required.
+
+Specifies a **PSCredential** object that this cmdlet use to connect to the NDES role service. To
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md:173]:
```suggestion
Specifies a **PSCredential** object that this cmdlet uses to connect to the NDES role service. To
```

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md [hunk 3]
```diff
@@ -395,14 +456,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### Microsoft.CertificateServices.Deployment.Common.NDES.NetworkDeviceEnrollmentServiceResult
 
 ## NOTES
-* Ensure you run Windows PowerShell as an administrator. You can use the *Force* parameter to bypass the prompt for confirmation.
-To see parameters, run the following command: `Install-AdcsNetworkDeviceEnrollmentService -?`
 
-  
+- Ensure you run Windows PowerShell as an administrator. You can use the **Force** parameter to bypass
+  the prompt for confirmation. To see parameters, run the following command:
+  `Install-AdcsNetworkDeviceEnrollmentService -?`
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md:462]:
```suggestion

    `Install-AdcsNetworkDeviceEnrollmentService -?`
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Snozzberries, minor suggested changes, please. Many thanks.

**Comment by Snozzberries** [comment]
Updated with changes.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md [hunk 4]
```diff
@@ -48,42 +57,70 @@ Int is equivalent to Int32 in the [.NET Framework](https://msdn.microsoft.com/en
...
+    WhatIf                 = $true
+}
+Install-AdcsNetworkDeviceEnrollmentService @params
 ```
 
-This command displays the default settings when NDES is using a service account without making any changes to the configuration.
-This command uses the service account named "CONTOSO\svcNDES" that is a member of the local computer's IIS_USRS group.
+This command displays the default settings when NDES is using a service account without making any
+changes to the configuration. This command uses the service account named `"CONTOSO\svcNDES"` that
+is a member of the local computer's `IIS_USRS` group.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md:80]:
```suggestion
This command displays the default settings when NDES is using a service account without making any
changes to the configuration. This command uses the service account named `CONTOSO\svcNDES` that
is a member of the local computer's `IIS_USRS` group.
```

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md [hunk 5]
```diff
@@ -48,42 +57,70 @@ Int is equivalent to Int32 in the [.NET Framework](https://msdn.microsoft.com/en
...
+    EncryptionProviderName = "Microsoft Strong Cryptographic Provider"
+    EncryptionKeyLength    = 4096
+}
+Install-AdcsNetworkDeviceEnrollmentService @params
 ```
 
-This command installs the NDES using a service account named "CONTOSO\svcNDES" that is a member of the local computer's IIS_USRS group.
-The command also specifies several non-default parameters.
+This command installs the NDES using a service account named `"CONTOSO\svcNDES"` that is a member of
+the local computer's `IIS_USRS` group. The command also specifies several non-default parameters.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md:115]:
```suggestion
This command installs the NDES using a service account named `CONTOSO\svcNDES` that is a member of
the local computer's `IIS_USRS` group. The command also specifies several non-default parameters.
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Minor suggestions around the specific values from the examples - we don't want to include the wrapping quotes when we format them as code.

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md [hunk 6]
```diff
@@ -48,42 +57,70 @@ Int is equivalent to Int32 in the [.NET Framework](https://msdn.microsoft.com/en
...
 This command displays the default NDES settings that will be configured if it is installed.
 
 ### Example 2: Display the default NDES settings using a service account name and password
-```
-PS C:\> Install-AdcsNetworkDeviceEnrollmentService -ServiceAccountName "CONTOSO\svcNDES" -ServiceAccountPassword (read-host "Set user password" -assecurestring) -WhatIf
+
+```powershell
+$params = @{
+    ServiceAccountName     = "CONTOSO\svcNDES"
+    ServiceAccountPassword = (read-host "Set user password" -assecurestring)
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md:72]:
```suggestion
    ServiceAccountPassword = (Read-Host "Set user password" -AsSecureString)
```

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md [hunk 7]
```diff
@@ -48,42 +57,70 @@ Int is equivalent to Int32 in the [.NET Framework](https://msdn.microsoft.com/en
...
+common name for `<CAComputerName>` and `<CACommonName>`.
 
 ### Example 4: Install NDES using a specific service account
-```
-PS C:\> Install-AdcsNetworkDeviceEnrollmentService -ServiceAccountName "CONTOSO\svcNDES" -ServiceAccountPassword (read-host "Set user password" -assecurestring) -CAConfig "CAComputerName\CAName" -RAName "Contoso-NDES-RA" -RACountry "US" -RACompany "Contoso" -SigningProviderName "Microsoft Strong Cryptographic Provider" -SigningKeyLength 4096 -EncryptionProviderName "Microsoft Strong Cryptographic Provider" -EncryptionKeyLength 4096
+
+```powershell
+$params = @{
+    ServiceAccountName     = "CONTOSO\svcNDES"
+    ServiceAccountPassword = (read-host "Set user password" -assecurestring)
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsNetworkDeviceEnrollmentService.md:101]:
```suggestion
    ServiceAccountPassword = (Read-Host "Set user password" -AsSecureString)
```

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**